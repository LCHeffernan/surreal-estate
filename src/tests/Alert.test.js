import React from "react";
import { render } from "@testing-library/react";
import Alert from "../components/Alert";

describe("Alert", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Alert />);
    expect(asFragment()).toMatchSnapshot();
  });

//   xit("displays an error message", () => {
//     const { getByText } = render(<Alert message="Error!" />);
//     expect(getByText(/Error/).textContent).toBe("Error!");
//   });

//   xit("displays a success message", () => {
//     const { getByText } = render(<Alert message="Success!!!!" success />);

//     expect(getByText(/Success/).textContent).toBe("Success!!!!");
//   });
});
