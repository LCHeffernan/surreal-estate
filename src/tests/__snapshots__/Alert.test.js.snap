// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders correctly 1`] = `
<DocumentFragment>
  <div
    class="alert alert-error"
  />
</DocumentFragment>
`;
